src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/OffenderController.kt

```diff
@@ -105,4 +106,41 @@ class OffenderController(private val offenderService: OffenderService, private v
   ) {
     this.offenderService.updateProbationTeam(crn, body)
   }
+
+  @PutMapping(value = ["nomisid/{nomsId}/update-offender-details"])
```

Comment:
`nomsId` to match the other example? 

it should be `prisonNumber` but that's a whole 'nother kettle of fish

Reply:
`nomisId` is used more frequently in the UI, `nomsId` in the API 😅 I was flip-flopping back and forth on which to do, and am happy to change it to `nomsid` if you think it'd be better

Reply:
I think by example below I thought there was another endpoint that used it.. but I guess I meant the use in the pathVariable: "nomisid/{**nomsId**}"